book/src/design/proving-system/inner-product.md

```diff
@@ -3,8 +3,6 @@
 Halo 2 uses a polynomial commitment scheme for which we can create polynomial commitment
 opening proofs, based around the Inner Product Argument.
```

Comment:
I think we should update this doc file since it is `inner-product.md`.
We need the doc for `KZG` commitment scheme and its use.